### ISSUE TITLE
[feature request] Add export for classic traveller character

### ISSUE THREAD

**Issue by Freyj** [Freyj/Fluffy-Hero-Maker#117]
On the flask app it would be nice to have a simple button that allows for the export of characters created (and of the whole party for the classic traveller part)

For dnd5 it is currently blocked by the issue #5 because there is no proper save.

**Comment by Freyj** [comment]
(making a specific issue for dnd5)
